packages/devtools_app/lib/src/screens/memory/panes/allocation_profile/allocation_profile_table_view.dart

```diff
@@ -265,12 +269,28 @@ class _AllocationProfileTable extends StatelessWidget {
     _FieldExternalSizeColumn(heap: _HeapGeneration.oldSpace),
   ];
 
-  final AllocationProfileTableViewController allocationProfileController;
+  final AllocationProfileTableViewController controller;
+
+  @override
+  State<_AllocationProfileTable> createState() =>
+      _AllocationProfileTableState();
+}
+
+class _AllocationProfileTableState extends State<_AllocationProfileTable> {
+  late SortDirection _sortDirection;
+  late ColumnData<ClassHeapStats> _sortColumn;
```

Comment:
can you link to a bug to store these values in a TableController? or if one does not exist, can you create one? I believe this is now the second case where we are storing these value for a Table in the widget that creates the Table